src/auxinfo/output.rs

```diff
@@ -181,10 +179,24 @@ mod tests {
         // Duplicate one of the PIDs
         pids.push(pids[4]);
 
-        let output = Output::simulate(&pids, rng);
+        let (mut private_auxinfo, public_auxinfo): (Vec<_>, Vec<_>) = pids
+            .iter()
+            .map(|&pid| {
+                let (key, _, _) = DecryptionKey::new(rng).unwrap();
+                (
+                    AuxInfoPrivate::from(key.clone()),
+                    AuxInfoPublic::new(
+                        &(),
+                        pid,
+                        key.encryption_key(),
+                        VerifiedRingPedersen::extract(&key, &(), rng).unwrap(),
+                    )
+                    .unwrap(),
+                )
+            })
+            .unzip();
```

Comment:
now that `simulate` checks that its output is valid, we have to manually create the input with duplicates.